Automattic/jetpack
# Jetpack: Compile TS extensions

Fix TS erros inside `extensions` folder in `Jetpack` plugin, and included `extensions` in build cycle/TS linting.

## Proposed changes:

* Remove specifics `tsconfig.json` files in extensions.
* Add `extensions` folder in Jetpack's tsconfig.
* Fix TS reported errors.
* Updated `global.d.ts`.

### Other information:


## Jetpack product discussion


p1707506855669229-slack-CBG1CP4EN

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Go to `/plugins/jetpack` folder.
* Run `pnpm compile-ts`.
* No error should appear
